content/2018-09-25-newsletter-12.md

```diff
@@ -0,0 +1,83 @@
+
+* [Diego Barrios Romero] has released his [eeprom24x] crate, an `embedded-hal` compatible driver for EEPROM memory chips such as the Microchip/Atmel `AT24C32`, `24AA32A`, `AT24C256`, or STMicroelectronics' `M24C32`
+* [Trangar] managed to drive [56 ArtNet Torches] using Rust, based on his [ArtNet Protocol] crate
+* The `no_std` synthesizer from the last newsletter has been implemented on-hardware by [Jonathan Pallant], giving the [Monotron] a [PWM audio driver]!
+
+[Diego Barrios Romero]: https://github.com/eldruin
+[eeprom24x]: https://blog.eldruin.com/24x-serial-eeprom-driver-in-rust/
+
+[Trangar]: https://github.com/trangar/
+[ArtNet Protocol]: https://github.com/Trangar/artnet_protocol
+[56 ArtNet Torches]: https://twitter.com/victorkoenders/status/1042786930335903745
+
+[Jonathan Pallant]: https://github.com/thejpster
+[Monotron]: https://github.com/thejpster/monotron
+[PWM audio driver]: https://twitter.com/therealjpster/status/1043942194015555584
+
+### `embedded-hal` Ecosystem Crates
+
+As part of the [Weekly Driver Initiative], crates that are part of the `embedded-hal` ecosystem are now tracked in the [Awesome Embedded Rust] repository. Here is a current snapshot of what is available there:
+
+| Type                      | Status    | Count | Diff |
+| :---                      | :-----    | :---- | :--- |
+| [Device Crates]           | released  | 16    | 0    |
+| [HAL Impl Crates]         | released  | 12    | 0    |
+| [Board Support Crates]    | released  | 9     | 0    |
+| [Driver Crates Released]  | released  | 12    | +1   |
+| [Driver Crates WIP]       | WIP       | 40    | -1   |
+| [no-std crates]           | released  | 13    | 0    |
+
+[Awesome Embedded Rust]: https://github.com/rust-embedded/awesome-embedded-rust
+[Weekly Driver Initiative]: https://github.com/rust-embedded/wg/issues/39
+[Device Crates]: https://github.com/rust-embedded/awesome-embedded-rust#device-crates
+[HAL Impl Crates]: https://github.com/rust-embedded/awesome-embedded-rust#hal-implementation-crates
+[Board Support Crates]: https://github.com/rust-embedded/awesome-embedded-rust#board-support-crates
+[Driver Crates Released]: https://github.com/rust-embedded/awesome-embedded-rust#driver-crates
+[Driver Crates WIP]: https://github.com/rust-embedded/awesome-embedded-rust#wip
+[no-std crates]: https://github.com/rust-embedded/awesome-embedded-rust#no-std-crates
+
+## Help Wanted
+
+* Take a minute to check out the [wg help wanted issues] as labeled on our issue tracker!
+* Interested in implementing a new target for Embedded Rust? Nuvoton has a [development board] for the Cortex-M23. Check out the [thumbv8m tracking issue] if you want to help!
+
+[wg help wanted issues]: https://github.com/search?q=org%3Arust-embedded+is%3Aissue+is%3Aopen+milestone%3Arc1+label%3A%22help+wanted%22&type=Issues
```

Comment:
This link returns an empty list. The milestone has been renamed from rc1 to edition, but there are no help-wanted edition issues either :-).

We could either remove this or link to *all* help-wanted issues within the org but that includes repos like cross, which I'm probably not to going to look at until next year.

Reply:
Will remove, :tada: :tada: :tada: !